packages/trust/src/extension/registry.ts

```diff
@@ -0,0 +1,22 @@
+import { Wallet } from '@cosmos-kit/core';
+import { GoDesktopDownload } from 'react-icons/go';
+import { RiChromeFill } from 'react-icons/ri';
+
+export const trustExtensionInfo: Wallet = {
+  name: 'trust-extension',
+  logo: 'https://trustwallet.com/assets/images/media/assets/TWT.png',
+  prettyName: 'Trust',
+  mode: 'extension',
+  mobileDisabled: true,
+  rejectMessage: {
+    source: 'Request rejected',
+  },
+  downloads: [
+    {
+      device: 'desktop',
```

Comment:
Can we include App Store/Google Play links here?

Reply:
not in this one, those would be for the mobile wallet :)